charts/jenkins/values.yaml

```diff
@@ -116,6 +116,12 @@ controller:
   #   runAsNonRoot: true
   #   supplementalGroups: [1000]
   #   # capabilities: {}
+  # Container securityContext
+  containerSecurityContext:
+    runAsUser: 1000
+    runAsGroup: 1000
```

Comment:
These two fields are also defined somewhere else. Maybe we could use `tpl` function when rendering it and reference the other values here.

https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-tpl-function
https://austindewey.com/2021/02/22/using-the-helm-tpl-function-to-refer-values-in-values-files/

Reply:
I double checked. Let's stick to your proposal as the other fields I mentioned are deprecated.